enhancements/windows-containers/health-management.md

```diff
@@ -51,6 +52,7 @@ and the node upgrade path from any WMCO version to another is codified through C
   + Container runtime
   + Metrics collector/exporter
 * Generate events to inform cluster administrators when Node health issues are not resolvable by WMCO.
+* Ensure the WICD watched environment variable configurations are set as expected on the Windows instances.
```

Comment:
You are introducing a goal without any context. This needs to be state more generically. Something to the effect of:
```suggestion
* Ensure environment variables defined by WMCO are set as expected on the Windows instances.
```

Add a couple of lines about this under motivation also.